src/config.rs

```diff
@@ -184,6 +206,28 @@ fn default_background() -> Rgba<f32> { Rgba {r: 0.0, g: 0.0, b: 0.0, a: 0.0} }
 mod tests {
     use super::*;
 
+    #[test]
+    fn resolve_absolute_path() {
+        let base_path = "/spritec/sample/bigboi";
+        let input_path = "/spritec/sample/bigboi/test";
+        let output_path = "/spritec/sample/bigboi/test";
+        resolve_check(base_path, input_path, output_path);
+    }
+
+    #[test]
+    fn resolve_relative_path() {
+        let base_path = "/spritec/sample/bigboi";
+        let input_path = "../../src/bin";
+        let output_path = "/spritec/src/bin";
+        resolve_check(base_path, input_path, output_path);
+    }
+
+    fn resolve_check(base: &str, input: &str, output: &str) {
```

Comment:
If you're curious, I can show you how we can use a Rust macro instead of this function to improve the tests and improve the output when tests fail. 

It's only a slight quality of life improvement, so it's definitely not necessary, but it might be fun to see if you're curious. :)

Reply:
I have been taught by the master